examples/metrics/prometheus-install/strimzi-pod-monitor.yaml

```diff
@@ -60,7 +60,7 @@ spec:
     matchExpressions:
       - key: "strimzi.io/kind"
         operator: In
-        values: ["Kafka", "KafkaConnect"]
+        values: ["Kafka", "KafkaConnect", "KafkaConnectS2I", "KafkaMirrorMaker2"]
```

Comment:
Why not Mirror Maker 1?

Reply:
@Frawless ^

Reply:
I thought mm1 doesn't have metrics.

Reply:
Documentation says the same

Reply:
I do not see any example resource deploying MM1 with metric, neither I can see any MM1 dashboard. So maybe we are just excluding it from the examples?

Reply:
It has the metrics field in the CRD. And I'm sure there will be at least producer and consumer metrics. It just doesn't have dashboard.

Reply:
In that case, we should add it and mention in docs (if I am not blind)

Reply:
Well, it is in the docs: https://strimzi.io/docs/operators/latest/full/using.html#type-KafkaMirrorMakerSpec-reference ... if we do not have dedicated examples and dashabord it might not need to be mentioned somewhere else but can be maybe still scraped if users enable the metrics with their configuration?

Reply:
So what about this one? Can you add the MM1 there?